kuma/payments/jinja2/payments/payments.html

```diff
@@ -224,10 +225,11 @@ <h2>{{_('Monthly Payments')}}</h2>
                         {% endtrans %}
                     </p>
                 {% endcall %}
-                {%- call faq_entry(14, _("Does deleting my MDN account delete my subscription?")) %}
+                {%- call faq_entry(14, _("Does deleting my MDN account cancel my subscription?")) %}
                     <p>
                         {% trans %}
-                            When you request to cancel your account we will also delete your subscription
+                        When you request to delete your account we will also cancel your subscription
+                        and not charge you for the following month.
```

Comment:
I prefer "not charge you for the following month", but maybe should be "not charge you for subsequent months" to match other phrases.

Reply:
Our lawyer confirms, will change to "subsequent months".